device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/MediaDto.kt

```diff
@@ -0,0 +1,7 @@
+package com.badoo.automation.deviceserver.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+class MediaDto(
+    @JsonProperty("media")
+    val url: String)
```

Comment:
property name is 'url' , in fact it is a string with file names, that are delimited by 'newline' symbol.

1. let's rename the property to something more relevant.
2. let's return a list of file names - i.e. actually a list of strings, so clients do not have to guess anything.